api/resolvers/user.js

```diff
@@ -44,7 +45,7 @@ async function authMethods (user, args, { models, me }) {
 
   return {
     lightning: !!user.pubkey,
-    email: user.emailVerified && user.email,
+    email: !!(user.emailVerified && user.emailHash),
```

Comment:
Convert email to a bool instead of showing the email value, which we no longer have.